src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/BookingExtractionService.kt

```diff
@@ -388,7 +389,7 @@ class BookingExtractionService(
     dates: MutableMap<ReleaseDateType, LocalDate>,
     latestReleaseDate: LocalDate,
     sentenceGroups: List<List<CalculableSentence>>,
-  ) {
+  ): Boolean {
```

Comment:
Maybe update this name as it's not clear what the boolean is returning without looking at the call site.

Reply:
The name is updated to reflect what the function does - extractErsedAndNotApplicableDueToDtoLaterThanCrdFlag